src/platform/unittest/TlsTest.cpp

```diff
@@ -96,6 +96,9 @@ struct TlsTest : public ::testing::TestWithParam<bool>
                 NULL,
                 QUIC_ALLOWED_CIPHER_SUITE_NONE};
             CredConfig.Flags |= CredFlags;
+#ifndef _WIN32
+            CredConfig.Flags |= QUIC_CREDENTIAL_FLAGS_USE_PORTABLE_CERTIFICATES;
+#endif
```

Comment:
I'd rather have explicit testing for both scenarios and not just have each platform test something else. Also, the code below (`OnPeerCertReceived`, which doesn't run for kernel) should be able to validate that both the cert and chain are non-null. Can you please make those updates?

Reply:
Updated. I added new test guarded by QUIC_DISABLE_PORTABLE_CERTIFICATE_TESTS and I added non-null check. 

I also look and the `SecPkgContext_Certificates` and it is actually the whole certificate chain. So I decided to pass same pointer as `Chain` instead of keeping it NULL.